src/Appium.Net/Appium/AppiumDriver.cs

```diff
@@ -279,13 +279,12 @@ public void Rotate(Dictionary<string, int> opts)
 
         public void BackgroundApp() =>
             Execute(AppiumDriverCommand.BackgroundApp,
-                AppiumCommandExecutionHelper.PrepareArgument("seconds",
-                    AppiumCommandExecutionHelper.PrepareArgument("timeout", null)));
+                AppiumCommandExecutionHelper.PrepareArgument("seconds", -1));
 
         public void BackgroundApp(int seconds) =>
             Execute(AppiumDriverCommand.BackgroundApp,
                 AppiumCommandExecutionHelper.PrepareArgument("seconds",
-                    AppiumCommandExecutionHelper.PrepareArgument("timeout", seconds)));
+                    TimeSpan.FromSeconds(seconds).TotalSeconds));
```

Comment:
I don't see any sense in converting the value from seconds to seconds. It would probably make more sense if the type of the method argument itself would be TimeSpan

Reply:
I would love to change the signature to Timespan, but didn't want to break the API, especially as it'll go in a minor release.

```Timespan.FromSeconds.TotalSeconds``` just sends a double

I could just send the raw int. What do you think @mykola-mokhnach

Reply:
I'm tempted to break it, but since you've unified the type to seconds server side, I'll keep it as int, and remove the redundant timespan

Reply:
The usual approach is to make this method signature deprecated and just add another one with the correct argument. Then we could remove the deprecated method in the next major release

Reply:
Cool. I can do that